psalm.xml

```diff
@@ -27,6 +27,7 @@
 		<UndefinedClass>
 			<errorLevel type="suppress">
 				<referencedClass name="OC\Hooks\Emitter" />
+				<referencedClass name="Doctrine\DBAL\Types\Type" />
```

Comment:
you should not use this. please migrate to `OCP\DB\Types` instead :)

Reply:
ah forgot to remove that from previous code that changed column's type.